sites/kit.svelte.dev/src/lib/docs/Contents.svelte

```diff
@@ -18,16 +18,13 @@
 
 	onMount(() => {
 		// wait for fonts to load...
-		const timeouts = [setTimeout(onresize, 1000), setTimeout(onscroll, 5000)];
-
-		update();
-		highlight();
+		const timeout = setTimeout(() => {
+			update();
+			highlight();
+		}, 1000);
```

Comment:
does this mean there won't be any contents until 1 second after load? I don't really understand why we need these function calls both here an in `afterNavigate`

Reply:
the comment above explains. it's there so that positions are correctly calculated if fonts are slow to load

Reply:
updated to use `document.fonts.ready`. Preloading is probably a step too far — would be finicky to get the URL to preload (unless Vite provided it to us), and means that other resources are deprioritised